usermods/Animated_Staircase/Animated_Staircase.h

```diff
@@ -480,10 +484,10 @@ class Animated_Staircase : public Usermod {
             (oldBottomAPin != bottomPIRorTriggerPin) ||
             (oldBottomBPin != bottomEchoPin)) {
           changed = true;
-          pinManager.deallocatePin(oldTopAPin);
-          pinManager.deallocatePin(oldTopBPin);
-          pinManager.deallocatePin(oldBottomAPin);
-          pinManager.deallocatePin(oldBottomBPin);
+          DEALLOCATE_PIN(oldTopAPin, PinOwner::UM_AnimatedStaircase);
```

Comment:
PinManager APIs treat attempts to deallocate pin `-1` as a no-op.  Simplifies client cleanup.